cg/cli/set.py

```diff
@@ -98,40 +121,84 @@ def samples(context, identifiers, kwargs, skip_lims, yes):
         )
 
 
+def list_changeable_sample_attributes(sample_obj: models.Sample = None, skip_list: list() = None):
+    """ list changeable attributes on sample """
+    keys = sample_obj.__dict__.keys() if sample_obj else models.Sample.__dict__.keys()
+    for key in keys:
+        if key.startswith("__") or key.startswith("_") or key in skip_list:
+            continue
+
+        message = key
+
+        if sample_obj:
+            message += f": {sample_obj.__dict__.get(key)}"
+
+        click.echo(message)
+
+
+def show_set_sample_help(sample_obj: models.Sample = "None"):
+    """Show help for the set sample command"""
+    show_option_help(option_long=OPTION_LONG_SKIP_LIMS, option_help=HELP_SKIP_LIMS)
+    show_option_help(OPTION_SHORT_YES, OPTION_LONG_YES, HELP_YES)
```

Comment:
Should you not use named args throughout?

Reply:
I needed to name them since I wanted to skip one. Do you think it is better practice to name all of them?

Reply:
fixed

Reply:
I think so! For consistency